frontend/packages/dev-console/integration-tests/features/getting-started/getting_started_tour_dev_perspective.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Getting Started tour of developer perspective
+    As a user I want to take tour of developer perspective
+
+
+@regression
+Scenario: Quick tour when user logs in for the first time
+   Given user is on login page for the first time
+   When user enters the credential
+   And user clicks on the login button
+   And user switches to developer perspective if not already there
+   And the Getting Started experience is automatically displayed
+   And user can see first screen as 'Welcome to Dev perspective' 
+   And user can see the button labels on the first screen are 'Get Started' & 'Close'
+   And user clicks on Get started 
+   And user can see next screen appears on Perspective switcher
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Monitoring tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Search tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Command Line Terminal
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Help
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears is Final Screen saying 'You are ready to go!'
+   And user can see the button labels as 'Got it' & 'Back to tour'
+   And user can clicks on 'Got it' button
+   Then user is taken to Add view in developer perspective
```

Comment:
guided tour doesn't redirect user to any page. user will be on the same page he started the tour on. By default, we load the topology page. So the user will be on topology page not add page.